R/scale-date.r

```diff
@@ -125,7 +125,10 @@ scale_datetime <- function(aesthetics, trans,
     minor_breaks <- date_breaks(date_minor_breaks)
   }
   if (!is.waive(date_labels)) {
-    labels <- date_format(date_labels)
+    labels <- function(self, x) {
+      tz <- if (is.null(self$timezone)) "UTC" else self$timezone
```

Comment:
How should this behave if there is a mismatch between the timezone in the data and the one provided in the constructor? Should the constructor value only kick in if the data does not have a timezone provided?